src/js/reducers/sessionReducer.jsx

```diff
@@ -49,6 +51,7 @@ const initialState = {
   minimumExpirationDate: '',
   isPaginated: false,
   logoLabel: '',
+  modal: false,
```

Comment:
isOpen (or isLocationChooserOpen to indicate what modal is open) sounds better for boolean

Reply:
Done